lib/elasticsearch/indexing/bulk.ex

```diff
@@ -121,7 +121,18 @@ defmodule Elasticsearch.Index.Bulk do
   end
 
   defp put_bulk_page(config, index_name, items) when is_list(items) do
-    Elasticsearch.put(config, "/#{index_name}/_doc/_bulk", Enum.join(items))
+    case Elasticsearch.put(config, "/#{index_name}/_doc/_bulk", Enum.join(items)) do
+      {:ok, response} ->
+        {:ok, response}
+
+      {:error, elasticsearch_exception} ->
+        if Map.get(config, :enable_debug_bulk_put) do
+          Logger.error("put_bulk_page error: #{inspect(elasticsearch_exception, limit: :infinity)}")
+          Logger.info("put_bulk_page errored items: #{inspect(items, limit: :infinity)}")
```

Comment:
This is going to be a list of potentially 2,000 fairly large maps. I don't think it will render in DD. Should we just log the IDs instead? Or maybe one log line per item?

Reply:
I like the idea of adding a log per item